Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -541,12 +541,25 @@ public static partial class PythonOps {
             return multiplier(sequence, icount);
         }
 
+        internal static object TryCoerce(CodeContext/*!*/ context, object x, object y) {
+            PythonTypeSlot pts;
+            PythonType xType = DynamicHelpers.GetPythonType(x);
+
+            if (xType.TryResolveSlot(context, "__coerce__", out pts)) {
+                object callable;
+                if (pts.TryGetValue(context, x, xType, out callable)) {
+                    return PythonCalls.Call(context, callable, y);
+                }
+            }
+            return NotImplementedType.Value;
+        }
+
         internal static int GetSequenceMultiplier(object sequence, object count) {
             int icount;
             if (!Converter.TryConvertToIndex(count, out icount)) {
                 PythonTuple pt = null;
                 if (count is OldInstance || !DynamicHelpers.GetPythonType(count).IsSystemType) {
-                    pt = Builtin.TryCoerce(DefaultContext.Default, count, sequence) as PythonTuple;
+                    pt = TryCoerce(DefaultContext.Default, count, sequence) as PythonTuple;
```

Comment:
Yes, it is gone, but I was going to remove it as part of the removal of OldClass and friends.